flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -38,13 +38,18 @@ message CreateTaskRequest {
   // The inputs required to start the execution. All required inputs must be
   // included in this map. If not required and not provided, defaults apply.
   // +optional
-  core.LiteralMap inputs = 1;
+  // Deprecated: Use inputs instead.
+  core.LiteralMap deprecated_inputs = 1 [deprecated = true];
   // Template of the task that encapsulates all the metadata of the task.
   core.TaskTemplate template = 2;
   // Prefix for where task output data will be written. (e.g. s3://my-bucket/randomstring)
   string output_prefix = 3;
   // subset of runtime task execution metadata.
   TaskExecutionMetadata task_execution_metadata = 4;
+  // Inputs are the inputs required to start the execution. All required inputs must be
+  // included in this map. If not required and not provided, defaults apply.
+  // +optional
+  core.InputData inputs = 5;
```

Comment:
will this break existing agents?  so you'll need to upgrade all agents to read this new field when deploying this change?